### ISSUE TITLE
Editor Goes Into Fits

### ISSUE THREAD

**Issue by gentisle** [mucommander/mucommander#724]
Version: 1.0.1
Build date: 2022/07/06
Build number: f49e48b

Java
Runtime version: 17.0.1
VM name: OpenJDK 64-Bit Server VM
VM version: 17.0.1+12
VM vendor: Eclipse Adoptium

OS
Name: Mac OS X
Version: 10.15.7
Architecture: x86_64

Locale
Language: en
Country: US
Encoding: UTF-8

When I try to edit a certain text file, the program brings up the find dialog asking for input. I cannot do anything with it. Pressing escape does nothing. The program refuses to give back control. I have to terminate muCommand via the menu. Then shutdown. The reason for shutdown is because the mouse cursor is blinking fast, and the system has become weird-responsive. This also occurred when trying to copy a .deb file from a thumb drive.

**Comment by ahadas** [comment]
@gentisle IIUC it's something that reproduces on your machine, right? if so. could you please provide logs / recording of the screen or something else can could help us invesgiate / reproduce this issue?

**Comment by gentisle** [comment]
OK, so here is the screen shot. Just the dialog that comes up and keeps me from doing anything. Also, I noted that the F3 key was light, and constantly on as if someone had laid a sledge hammer on it. I could not get F3 to release by pressing any combo of keys.  It does do it on other text files. I will say that I have the top row of my Mac set to be the 'F' Keys always. Also, the file was edited in Windows, but my Windows editor allows me to save text files with Unix line enders. So I cannot figure why it would freak muCommander our like that. But since it also did it on a file copy...??? I am wondering if it has a scrambled install from when there was another bug, and I tried uninstalling/re-installing an earlier version, which failed. Apple's OS' doesn't seem to allow for complete uninstalling except in certain cases.
Sorry, I cannot show you the file because it has sensitive data.
gentisle
<img width="267" alt="Screen Shot 2022-09-03 at 18 12 31" src="https://user-images.githubusercontent.com/46111092/188293042-a21f3ce4-b6fb-4b50-a596-f7bc6004b119.png">

**Comment by gentisle** [comment]
I just noticed that you asked for logs, but I don't know where muCommander stores logs on MacOS. Also, this morning I remembered that I have Nimble Commander and Commander One, and I tried the latter, with no problem.  Also, I cannot reproduce the error today, perhaps it was a holiday error?? (sorry, bad joke). But I would provide the logs, if I could find them.

**Comment by ahadas** [comment]
@gentisle yeah, a screenshot of the 'Find' dialog doesn't give much :) the log file(s) appears within the `logs` folder under the [Preferences folder](https://github.com/mucommander/mucommander/wiki/Preference-Files). this issue is unlikely to be an application issue though so I'm closing it for now and if that problem happens again please reopen and provide the logs so we could see what the application gets, maybe that would help identifying the root cause

**Comment by gentisle** [comment]
Yeah, I think I found the issue, but not sure. Closed is fine. Thanks. There was an app that I have given permission to read/write - full disk access. When I took that away, the problem with muCommander seemed to disappear. Freaky! Yes, sorry about that graphic, but that just popped up over the muCommander editor, and would let me have control, so I had to kill the program from the Apple menu. There was nothing else to show, sorry. Don't know why another program's permission would do that, but then millions of people worked on the Macbook and MacOS, and all the hardware/software, and they all didn't get to confer about it. In other words, nobody is God who can think of everything, so we all make mistakes and computers have errors. (I think I make more than most.) Thanks again for your help.
gentisle
